src/Pdf/Styles/BorderStyle.cs

```diff
@@ -1,4 +1,4 @@
-namespace InvoiceKit.Pdf;
+namespace InvoiceKit.Pdf.Styles;
```

Comment:
I don't like moving these into a Styles namespace because everything needs to use the Styles namespace. I believe styling should be a part of the root PDF project the same as `ILayout` or `PdfDocument`